app/src/main/java/edu/usf/sas/pal/muser/util/FirebaseIOUtils.java

```diff
@@ -72,21 +80,55 @@ private static synchronized String getRecordID(){
 
     public static void logErrorMessage(Exception e, String message) {
         if (e != null) {
-            Log.d(TAG, message + e.getMessage());
+            Log.d(TAG, message +
```

Comment:
This can all be on one line

Reply:
I guess most of these changes are from the `register-user` branch cause I remember fixing them in the `firebase` branch